packages/teleport/src/Console/DocumentSsh/DocumentSsh.tsx

```diff
@@ -64,14 +62,6 @@ export default function DocumentSsh({ doc, visible }: PropTypes) {
             <Indicator />
           </Box>
         )}
-        {status === 'error' && (
-          <Alerts.Danger mx="10" mt="5">
-            Connection error: {statusText}
-          </Alerts.Danger>
-        )}
-        {status === 'notfound' && (
-          <SidNotFoundError sid={doc.sid} clusterId={doc.clusterId} />
-        )}
```

Comment:
if i enter an invalid SID in the URL, my terminal just says `disconnected` (which is not helpful to the user). Before we use to query for the existence of session ID first and if an error returns, we would assume the session is no longer active, which may or may not be true (which can cause confusion if it wasn't true, bad wording). If it was true though, then user can be directed to the recording.

Can we send back a more user friendly error message from the back, since we no longer query if the session exists before attempting wss cn?

Reply:
We definitely can do. I added a `WebsocketError` [message type](https://github.com/gravitational/teleport/pull/18523/files#diff-cc34e8b8bef47626c4518b46eb5a56458acedd961d290d5a79b5fe5ff63fa832R701) in the backend for other failures but didn't send one for this case. I'll make a note to add one in a follow-up.